src/main/kotlin/ar/edu/unahur/obj2/vendedores/CentroDistribucion.kt

```diff
@@ -0,0 +1,21 @@
+package ar.edu.unahur.obj2.vendedores
+
+class CentroDistribucion(val ciudad: Ciudad) {
+    val vendedores = mutableListOf<Vendedor>()
+
+    fun agregarVendedor(vendedor: Vendedor) {
+        if (! vendedores.contains(vendedor)) {
+            vendedores.add(vendedor)
+        } else {
+            error("El vendedor ya se encuentra registrado en este Centro.")
+        }
```

Comment:
Está bien, aunque queda un poco más prolijo poner el error en la rama verdadera del `if` y obviar el `else`:
```suggestion
        if (vendedores.contains(vendedor)) {
            error("El vendedor ya se encuentra registrado en este Centro.")
        }
        vendedores.add(vendedor)            
```